pages/index.html

```diff
@@ -28,11 +28,16 @@
       <div class="navbar-header">
         <a href="index.html" class="navbar-brand"><h1>Libook</h1></a>
       </div>
+      <div class="row">
       <form action="#" class="form-inline">
-        <input id="search-text" name= "search-text" type="text" class="form-control" placeholder="Search">
-        <button id="search-btn" class="btn btn-outline-success ml-2" type="submit">Search</button>
+     
+        <div class=" col-xs-3 " > <input id="search-text" class="search-small" name= "search-text" type="text" class="form-control" placeholder="Search"></div>
```

Comment:
here the class has been changed from form-control to search-small. Try using 'search-small form-control'
![image](https://user-images.githubusercontent.com/47845897/72840495-5a9c5580-3cba-11ea-9ccf-5850b5b49870.png)